public/css/about-teams.css

```diff
@@ -0,0 +1,97 @@
+* {
+    margin: 0px;
+    padding: 0px;
+}
+
+.about-teamsBody {
+    margin: 0px;
+    background-color: var(--greyBgC);
+    color: var(--whiteFontC);
+ }
+
+ body > .logoDiv {
+     position: static;
+     width: 100%;
+     margin-bottom: 10vh;
+ }
+
+ h1 {
+    font-family: var(--rajdhaniFont);
+    font-weight: normal;
```

Comment:
Everything looks amazing! Just one tiny thing, change the text here to numbers instead. Here it should be 500. :D

Reply:
Requested changes has been implemented.